jetty-server/src/main/java/org/eclipse/jetty/server/ResourceService.java

```diff
@@ -278,6 +284,61 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         }
     }
 
+    private CompressedContentFormat getBestPrecompressedContent(String accept, Collection<CompressedContentFormat> availableFormats) {
+        if (accept == null || availableFormats.isEmpty())
+        {
+            return null;
+        }
+        CompressedContentFormat bestEncoding = null;
+        float bestResourceQuality=0;
+        for (String preference:accept.split(","))
```

Comment:
This code should be replaced with the recently added utility class [QuotedQualiytCSV](https://github.com/eclipse/jetty.project/blob/master/jetty-http/src/main/java/org/eclipse/jetty/http/QuotedQualityCSV.java).